src/instructlab/configuration.py

```diff
@@ -28,6 +28,8 @@
 DEFAULT_CONFIG = "config.yaml"
 # TODO: Consolidate --model and --model-path into one --model-path flag since we always need a path now
 DEFAULT_MODEL_OLD = "merlinite-7b-lab-Q4_K_M"
+DEFAULT_REPOSITORY = "instructlab/merlinite-7b-lab-GGUF"
```

Comment:
we have a few repos.  How about naming similar to the taxonomy repo below.  That would also give these a nice consistent "DEFAULT_MODEL_" prefix.  So.

```suggestion
DEFAULT_MODEL_REPO = "instructlab/merlinite-7b-lab-GGUF"
```

Reply:
6113b74